iptables/iptables.go

```diff
@@ -107,7 +108,8 @@ func NewChain(name, bridge string, table Table, hairpinMode bool) (*Chain, error
 		}
 		output := []string{
 			"-m", "addrtype",
-			"--dst-type", "LOCAL"}
+			"--dst-type", "LOCAL",
+			"-j", c.Name}
```

Comment:
Since the `Remove` method also uses the `Output()`, isnt it better to include the above `-j` target to all the callers of that existing API (Output and Prerouting) ?